Source/Core/Core/IOS/WFS/WFSI.h

```diff
@@ -32,8 +32,6 @@ class ARCUnpacker
   std::vector<u8> m_whole_file;
 };
 
-namespace Device
-{
 class WFSI : public Device
```

Comment:
Do we want to name this one `WFSIDevice` for consistency?

Reply:
Oh whoops, I forgot to rename this. I left the USB classes alone to avoid having everything called a "device" in the USB code (considering we have IOS devices and lots of USB devices there...) For WFS, I don't think there's any particular reason not to use the suffix.